src/fee-accounts/fee-accounts.routing.ts

```diff
@@ -0,0 +1,38 @@
+// routes
+import {RouterModule, Routes} from '@angular/router';
+import {ModuleWithProviders} from '@angular/core';
+import {FeeAccountsComponent, TransactionsComponent, SingleFeeAccountComponent} from './containers';
+
+export const ROUTES: Routes = [
+  {
+    path: 'fee-accounts',
+    children: [
+      {
+        path: '',
+        component: FeeAccountsComponent
+      },
+      {
+        path: 'account',
+        component: FeeAccountsComponent
+      },
+      {
+        path: 'account/:id',
+        component: SingleFeeAccountComponent,
+        children: [
+          {
+            path: 'summary',
+            component: SingleFeeAccountComponent
+          },
+          {
+            path: 'transactions',
+            component: SingleFeeAccountComponent
+          }
+
+        ]
+      }
+    ]
+  }
+];
```

Comment:
do we need so many routes

Reply:
yes

Reply:
Yes, because the sub navigation works on a route basis